sycl/test/gdb/iostream_test.cpp

```diff
@@ -0,0 +1,13 @@
+// UNSUPPORTED: true
+// RUN: %clangxx -fsycl -g %s -o %t.out
+// RUN: gdb %t.out -batch -x %sycl_gdb_iostream 2>&1 | FileCheck %s
+// CHECK: ignore next 9998 hits
+//
+// Tests if <sycl/sycl.hpp> headers include any <iostream> headers
```

Comment:
I think it can be done w/o running a GDB, but just checking the compiler output.
NOTE: it checks host side of the compilation only, so you don't need to compile with `-fsycl` option.